index.html

```diff
@@ -1366,34 +1366,44 @@ <h2>The <dfn>NDEFRecordType</dfn> enum</h2>
       </p>
 
       <pre class="idl">
-        enum NDEFRecordType {
+        enum NDEFKnownRecordType {
           "empty",
           "text",
           "url",
           "json",
           "opaque"
         };
+
+        typedef (NDEFKnownRecordType or DOMString) NDEFRecordType;
       </pre>
-      <dl>
+      <dl data-dfn-for="NDEFKnownRecordType">
         <dt><dfn>empty</dfn></dt>
         <dd>
-          The enum value representing <a href="#dfn-empty">empty</a> <a>NDEFRecord</a>.
+          The value representing <a href="#dfn-empty">empty</a> <a>NDEFRecord</a>.
         </dd>
         <dt><dfn>text</dfn></dt>
         <dd>
-          The enum value representing <a>NDEFRecord</a> of a <a href="#dfn-text">text</a> type.
+          The value representing <a>NDEFRecord</a> of a <a href="#dfn-text">text</a> type.
         </dd>
         <dt><dfn>url</dfn></dt>
         <dd>
-          The enum value representing <a>NDEFRecord</a> of a <a href="#dfn-url">url</a> type.
+          The value representing <a>NDEFRecord</a> of a <a href="#dfn-url">url</a> type.
         </dd>
         <dt><dfn>json</dfn></dt>
         <dd>
-          The enum value representing <a>NDEFRecord</a> of a <a href="#dfn-json">json</a> type.
+          The value representing <a>NDEFRecord</a> of a <a href="#dfn-json">json</a> type.
         </dd>
         <dt><dfn>opaque</dfn></dt>
         <dd>
-          The enum value representing <a>NDEFRecord</a> of an <a href="#dfn-opaque">opaque</a> type.
+          The value representing <a>NDEFRecord</a> of an <a href="#dfn-opaque">opaque</a> type.
+        </dd>
+        <dt><dfn>External type</dfn></dt>
```

Comment:
We may need add the external type to the [data mapping](https://w3c.github.io/web-nfc/#data-mapping) table.

Reply:
That is done, try the preview